app/src/androidTest/java/com/sdp/movemeet/utility/ActivitiesUpdaterTest.java

```diff
@@ -45,47 +46,4 @@ public void instanceIsNeverNull() {
         assertNotNull(updater);
     }
 
-    @Test
```

Comment:
So you deleted this test to move it to MainMapFragmentTest? But then is this ActivitiesUpdaterTest still useful since it is almost empty ?

Reply:
I intend to add some other tests in this class